docs/docs/integrations/providers/huggingface.mdx

```diff
@@ -26,6 +26,19 @@ To use the local pipeline wrapper:
 from langchain.llms import HuggingFacePipeline
 ```
 
+To use the OpenVINO backend in local pipeline wrapper, please install the optimum library and set HuggingFacePipeline's backend as `openvino`:
+`
+pip install --upgrade-strategy eager "optimum[openvino,nncf]"
```

Comment:
should we somewhere (requirements.txt or pyproject.toml file) add this dependency? (optional or extras)

Reply:
Yes, I also wonder if we can make it as a dependency somewhere.
Could you help to provide some suggestions? @baskaryan

Reply:
> should we somewhere (requirements.txt or pyproject.toml file) add this dependency? (optional or extras)

Just make it as a optional dependency in pyproject.toml file

Reply:
should we use poetry command instead of pip? 
I see poetry.lock file is updated

Reply:
> poetry

This is a good idea, but if looking at other dependency installation hint, you will find all of them are using ```pip install```, for example [huggingface pipeline](https://github.com/langchain-ai/langchain/blob/0019cee6ffa8595557e69885f0230cfc4ee6e2bb/docs/docs/integrations/providers/huggingface.mdx#L14). Another reason is that i would like to suggest developer to install latest version of Optimum which may support more model architecture.

Reply:
> should we use poetry command instead of pip? I see poetry.lock file is updated

This is a good idea, but if looking at other dependency installation hint, you will find all of them are using pip install, for example [huggingface pipeline](https://github.com/langchain-ai/langchain/blob/0019cee6ffa8595557e69885f0230cfc4ee6e2bb/docs/docs/integrations/providers/huggingface.mdx#L14). Another reason is that i would like to suggest developer to install latest version of Optimum which may support more model architecture.